taxonomies/data_quality.txt

```diff
@@ -2912,6 +3076,18 @@ en:Carbon footprint from known ingredients more than from meat or fish
 
 ### Labels
 
+<en:Data quality errors
+en:vegan-label-but-non-vegan-ingredient
+fr:étiquette végétalienne mais ingrédient non végétalien
+es:etiqueta vegana pero ingrediente no vegano
+description:en:This product has a vegan label but contains non-vegan ingredients.
+
+<en:Data quality errors
+en:vegetarian-label-but-non-vegetarian-ingredient
+fr:étiquette végétarienne mais ingrédient non végétarien
+es:etiqueta vegetariana pero ingrediente no vegetariano
+description:en:This product has a vegetarian label but contains non-vegetarian ingredients.
+
```

Comment:
Same as previous comment. This is error, it should be in the first part of the file around line 811

```suggestion
```